web/src/lib/roadmap/buildUserRoadmap.ts

```diff
@@ -21,7 +21,7 @@ export const buildUserRoadmap = async (profileData: ProfileData): Promise<Roadma
     industryId = "employment-agency";
   }
 
-  if (profileData.industryId === "interior-designer" && !profileData.certifiedInteriorDesigner) {
+  if (industryId === "interior-designer" && !profileData.certifiedInteriorDesigner) {
```

Comment:
Replaced `profileData.industryID` in a few places because the definition of `industryID` for this function already uses this.

Reply:
good refactor